reporting_api.md

```diff
@@ -0,0 +1,118 @@
+# Reporting API
+
+## Publisher Performance Report (JSON)
+
+### URL:
+```
+http://api.truex.com/v1/publisher/performance.json
+```
+
+### Parameters:
+
+- `api_key` (STRING) - The partner-specific reporting api key provided by true[X]
```

Comment:
how does one get an API key? talk to helpdesk or devops?

Reply:
It's provided by true[X] during integration I guess? Probably around the same time we give them placement hashes and the shared secret they use to validate signatures